config/settings.d/tftp.yml.example

```diff
@@ -5,3 +5,15 @@
 #:tftproot: /var/lib/tftpboot
 # Defines the TFTP Servername to use, overrides the name in the subnet declaration
 #:tftp_servername: tftp.domain.com
+
+# Defines the default read timeout in seconds needed to download tftp artifacts
+# like initrd and vmlinuz. Default value 60 seconds
+#:tftp_read_timeout: 60
```

Comment:
Could you add timeout settings and their values to the list of default settings here: https://github.com/theforeman/smart-proxy/blob/develop/modules/tftp/tftp_plugin.rb#L8

Reply:
@witlessbird done